ui/pages/import-token/import-token.component.js

```diff
@@ -524,7 +530,17 @@ class ImportToken extends Component {
         />
         <TextField
           id="custom-decimals"
-          label={t('decimal')}
+          label={
+            <span
+              className={
+                decimalAutoFilled
+                  ? 'import-token__custom-decimals__label--disabled'
+                  : 'import-token__custom-decimals__label'
+              }
+            >
+              {t('decimal')}
+            </span>
```

Comment:
I think i we are going to add the span here instead of using straight text, we should also add the wrapper similar to the other instance. The main role it has is adding a flex box i think.

Reply:
I understood the wrapper part, but I don't understand the adding span instead of straight text part.

Reply:
the wrapper part was was the important part, I just meant if we are going to use a react node instead of a string we need to include the wrapper as it was done on the other instance.

Reply:
Added the wrapper as well, let me know if it need further changes